mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/navigation/MapboxNavigationEvent.java

```diff
@@ -0,0 +1,169 @@
+package com.mapbox.services.android.telemetry.navigation;
+
+import android.location.Location;
+
+import com.mapbox.services.android.telemetry.constants.TelemetryConstants;
+import com.mapbox.services.android.telemetry.utils.TelemetryUtils;
+
+import java.util.Date;
+import java.util.Hashtable;
+
+/**
+ * Represent events dispatched by the Navigation SDK.
+ */
+public class MapboxNavigationEvent {
+
+  private static final int EVENT_VERSION = 1;
+
+  // Event types
+  public static final String TYPE_TURNSTILE = "navigation.turnstile";
+  public static final String TYPE_DEPART = "navigation.depart";
+  public static final String TYPE_FEEDBACK = "navigation.feedback";
```

Comment:
We've added a separate `navigation.reroute` event type in addition to `navigation.feedback`